cmd/feed-ingress/main.go

```diff
@@ -4,25 +4,82 @@ import (
 	"os"
 	"os/signal"
 	//
+	"flag"
+
+	"io/ioutil"
+
 	log "github.com/Sirupsen/logrus"
 	"github.com/sky-uk/feed/ingress"
 	"github.com/sky-uk/feed/k8s"
 )
 
+var (
+	apiServer  string
+	caCertFile string
+	tokenFile  string
+	debug      bool
+)
+
+func init() {
+	const (
+		defaultAPIServer  = "https://10.254.0.1:443"
```

Comment:
You want it to point to another cluster? Not sure what you mean. 10.254.0.1 is the apiserver vip inside of a kubernetes cluster, it shouldn't be access any other cluster.